engine/inc.settings.php

```diff
@@ -1,7 +1,16 @@
 <?php
 
+$options['default_language'] = 'en';
+$options['languages'] = array('en' => 'English', 'lv' => 'Latviešu', 'fr' => 'Français', 'ru' => 'Русский', 'nl' => 'Nederlands', 'pl' => 'Polski', 'es' => 'Spanish');
 
+$options['images']['small_width'] = 200;
```

Comment:
Why not just assign or extend an array here? I think that would make the configuration easier to read.